peps/pep-0722.rst

```diff
@@ -8,8 +8,13 @@ Type: Standards Track
 Topic: Packaging
 Created: 19-Jul-2023
 Post-History: `19-Jul-2023 <https://discuss.python.org/t/29905>`__
+Superseded-By: 723
 Resolution: https://discuss.python.org/t/pep-722-723-decision/36763/
 
+.. rejected::
+
+   This PEP was rejected in favour of :pep:`723`.
```

Comment:
To replace the default and remove duplication:

```suggestion
.. rejected:: This PEP was rejected in favour of :pep:`723`.
```
Before:

<img width="390" height="174" alt="Image" src="https://github.com/user-attachments/assets/b70f8e24-e28d-4439-b1ea-7050c9838ab4" />

After:

<img width="386" height="139" alt="Image" src="https://github.com/user-attachments/assets/01350734-e856-4e0a-94f2-9ddd38c99889" />